extensions/amp-animation/amp-animation.md

```diff
@@ -266,200 +307,265 @@ An example of timing properties in JSON:
+- Use [CSS `transform-box`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-box) to avoid `transform-origin` interpretation problems. See issues for [Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=740300), [Safari](https://bugs.webkit.org/show_bug.cgi?id=174285) and [Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1379340).
 
-- For broad-platform support, vendor prefixes, e.g. `@-ms-keyframes {}` or `-moz-transform` may be needed. Vendor prefixes are not needed and not allowed in the JSON format, but in CSS they could be necessary.
-- Platforms that do not support `calc()` and `var()` will not be able to take advantage of `amp-animation` polyfills when keyframes are specified in CSS. It's thus recommended to always include fallback values in CSS.
-- CSS extensions such as [`width()`, `height()`, `x()`, `y()`, `num()`, `rand()`, `index()` and `length()`](#css-extensions) cannot be used in CSS.
+### Compatibility and fallbacks
 
-#### White listed properties for keyframes
+Use media queries, support conditions and switch statements for platform compatibility and fallback options.
 
-Not all CSS properties can be used in keyframes. Only CSS properties that modern browsers can optimize and
-animate quickly are white listed. This list will grow as more properties are confirmed to provide good
-performance. Currently the list contains:
+#### Media queries
 
-- [`opacity`](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity)
-- [`transform`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform)
-- [`visibility`](https://developer.mozilla.org/en-US/docs/Web/CSS/visibility)
-- [`offset-distance`](https://developer.mozilla.org/en-US/docs/Web/CSS/offset-distance)
-- [`clip-path`](https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path). Only supported values are
-  `inset()`, `circle()`, `ellipse()`, and `polygon()`.
+Specify media queries with the `media` property. This property can contain any expression allowed
+for [Window.matchMedia](https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia) API and corresponds to `@media` CSS rule.
 
-Notice that the use of vendor prefixed CSS properties is neither needed nor allowed.
+When specified, the animation component will only appear when met with the defined environment.
 
-### Abbreviated forms of animation configuration
+#### Supports condition
 
-If the animation only involves a single element and a single keyframes effect is sufficient, the configuration
-can be reduced to this one animation component only. For instance:
+Specify supports conditions using the supports property. The support property contains any expression allowed for [CSS.supports](https://developer.mozilla.org/en-US/docs/Web/API/CSS/supports) API and corresponds to `@supports` CSS rule.
 
-```html
-<amp-animation layout="nodisplay">
-  <script type="application/json">
+When specified, the animation component will only appear when met with the defined environment.
```

Comment:
Not sure this reads clear. Maybe "... then animation component will only execute when the environment supports the specified CSS feature."?